diff --git a/node_modules/@capacitor/android/capacitor/build.gradle b/node_modules/@capacitor/android/capacitor/build.gradle
index caeeceb..06a37c3 100644
--- a/node_modules/@capacitor/android/capacitor/build.gradle
+++ b/node_modules/@capacitor/android/capacitor/build.gradle
@@ -64,8 +64,8 @@ android {
         lintConfig file('lint.xml')
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_21
-        targetCompatibility JavaVersion.VERSION_21
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
     publishing {
         singleVariant("release")
